.gitignore

```diff
@@ -0,0 +1,4 @@
+/src/Management/Carrello.json
```

Comment:
Per questo si potrebbe usare una wildcard, indicando tutti i file che finiscono per .json

Reply:
Per adesso però va bene, non è un problema critico in questa fase.

Reply:
Fatto!